cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/StreamHandler.java

```diff
@@ -180,6 +182,8 @@ public void create(HttpRequest request, HttpResponder responder,
       }
     }
 
+    streamLeaderManager.affectLeader(stream);
```

Comment:
Should I wait for the future to finish here? Also, I'm not sure what to do in case of failure of the future. Should be still be able to use the stream? Or return an error code, and roll-back the stream creation? It doesn't sound like a great idea, but we should still notify the user that the operation failed.